functions/src/controllers/gameController.ts

```diff
@@ -0,0 +1,24 @@
+import {Router} from "express";
+import {DelayUnities, Game, User} from "../bdd/entities";
+import {BddService} from "../services/BddService";
+import {BadRequestError} from "../errors";
+
+const router = Router();
+
+router.post("/create", async function(req, res, next) {
+    // TODO : Not functionnal 
+    const eliminationDelayUnity: DelayUnities = Game.castToDelayUnities(req.body.eliminationDelayUnity);
+    const eventIntervalUnity: DelayUnities = Game.castToDelayUnities(req.body.eventIntervalUnity);
+    const user = <User> await BddService.userHandler.findUserById(req.body.userId);
+    // const user = new User(["username": ])
+    if(user === null){
+        throw new BadRequestError("Invalid User Data");
+    }
+    let game;
+    game = await BddService.gameHandler.createGame(req.body,user,eventIntervalUnity, eliminationDelayUnity);
+    if (game != null) res.status(200);
+    else throw new BadRequestError("Invalid Game Data");
+});
+
```

Comment:
```suggestion
    game = await BddService.gameHandler.createGame(req.body,user,eventIntervalUnity, eliminationDelayUnity);
    
    if (game != null) res.status(200);
    else throw new BadRequestError("Invalid Game Data");
});

```

Tu peux définir ton let game directement ici ligne 18